src/schema/v2/user.ts

```diff
@@ -62,11 +77,34 @@ export const UserSaleProfileField: GraphQLFieldConfig<any, ResolverContext> = {
   },
 }
 
+export const UserAdminNotesField: GraphQLFieldConfig<any, ResolverContext> = {
+  description: "The admin notes associated with the user",
+  type: new GraphQLList(UserAdminNoteType),
+  resolve: ({ id }, {}, { userAdminNotesLoader }) => {
+    if (!userAdminNotesLoader) {
+      throw new Error(
+        "You need to be signed in as an admin to perform this action"
```

Comment:
Maybe should say "...need additional privileges to perform this action" in anticipation of the `admin` role not actually being sufficient (minor).

Actually, if this loader will exist (but not resolve) whenever the caller is authenticated, might be enough to say "...need to be signed in to perform this action."

Reply:
Yea, all MP does is make certain loaders inaccessible if a `X-Access-Token` header isn't passed in to MP as part of the request. Maybe it's even better to just say that instead? ("You need to pass an X-Access-Token header...")

Reply:
That makes sense. Will change the response to what is suggested.